sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -603,11 +603,16 @@ def add_file_mount(obj):
             for i, _ in enumerate(obj['spec']['containers']):
                 if 'volumeMounts' not in obj['spec']['containers'][i]:
                     obj['spec']['containers'][i]['volumeMounts'] = []
-                obj['spec']['containers'][i]['volumeMounts'].append({
-                    'name': cm_name,
-                    'mountPath': dst,
-                    'subPath': filename,
-                })
+                mount_list = obj['spec']['containers'][i]['volumeMounts']
```

Comment:
The current solution is wrong because it creates redundant resources in case of duplication and spams the pod configuration.

The proper solution would be to create a list of mounted files, look at it first, and if not found, then do all the 3 steps we have here:
- create config map
- attach it as a volume
- mount that volume to a file path.

And, if found (already mounted), do nothing.
Also, such approach assumes usage of a lock for whole method per class instance.